specification/dimensions/usagetype.md

```diff
@@ -0,0 +1,42 @@
+# Usage Type
+
+UsageType is a categorization parameter that defines how a resource or service was consumed during a specified period. The UsageType can influence the price point and also indicate whether purchased commitments are used or unused. This classification offers a nuanced view into the resource or service's consumption patterns and helps in understanding how you're billed, especially when considered alongside attributes like ChargeType and PurchaseType.
```

Comment:
> defines how a resource or service was consumed 
In case of unused commitment, there is no resource or service attached to this record (unless one considers the commitment is a resource, which is arguable), so referring a resource or service can be misleading. Maybe using the plural form works better: "defines how resources or services were used".

> during a specific period
Maybe "during the charge period" is more precise

> especially when considered alongside attributes like ChargeType and PurchaseType.
I understood that UsageType would be used to drilldown the type of the record in the case ChargeType=Usage. So UsageType would be populated (not empty, not null) if and only if ChargeType=Usage. In the same way, PurchaseType would be populated if and only if ChargeType=Purchase. Or would ChargeType, UsageType and PurchaseType work in another way?